samples/CommunityToolkit.Maui.Sample/MauiProgram.cs

```diff
@@ -46,6 +48,7 @@ public static MauiApp CreateMauiApp()
 #endif
 								.UseMauiCommunityToolkitMarkup()
 								.UseMauiCommunityToolkitMediaElement()
+								.UseMauiCommunityToolkitMaps("")
```

Comment:
Should we somehow note to the user that they have to provide their own here?

Reply:
the exception is thrown with instruction message if key is invalid

Reply:
This is what I get if I just pull down the code as is and don't enter a key. To me it's not really clear that this is about a missing key? 

![image](https://github.com/CommunityToolkit/Maui/assets/939291/139bc87d-4d9b-48c2-823b-3634539d437b)

Reply:
Yea, WinUI's Unhandled Exception reporting while debugging (as seen here) is laughably bad. It's a terrible developer experience.

Reply:
Maybe I have some different settings then or some other difference. Fine for now, let's deal with it when other people start complaining.